apps/angular/59-content-projection-defer/src/app/page-2.ts

```diff
@@ -36,8 +37,18 @@ interface Post {
   imports: [ExpandableCard],
 })
 export class Page2 {
+  protected readonly ResourceStatus = ResourceStatus;
+
   public postRessource = httpResource<Post[]>(
-    'https://jsonplaceholder.typicode.com/posts',
+    () => {
+      return this.isExpanded()
+        ? 'https://jsonplaceholder.typicode.com/posts'
+        : undefined;
+    },
+    {
+      defaultValue: [],
+    },
   );
-  protected readonly ResourceStatus = ResourceStatus;
+
+  isExpanded = signal(false);
```

Comment:
you can use `viewChild` or `exportAs` to access a child property as well. It creates less boilerplate, but your solution is working